config/locales/contact_us/contact_us.pt_BR.yml

```diff
@@ -1,25 +0,0 @@
-# Sample localization file for English. Add more files in this directory for other locales.
```

Comment:
We have a full translation for Brazillian due to their translation work with CDL so we'll actually want to keep and update this file rather than remove it.